### ISSUE TITLE
Dependents | Remove header from radio group

### ISSUE THREAD

**Issue by Mottie** [department-of-veterans-affairs/va.gov-team#110098]
## Issue Description

Within the current spouse's former marriage list loop, there is a question about how the previous marriage ended. The page already has an H3 ("Edit spouse's former marriage"). The radio group header is currently also an H3 ("How did your spouse's previous marriage end?") should either be an H4, or not a header at all. Given [the design doesn't have the radio label as a header](https://www.figma.com/design/7W55oNwdVXvXOTI9SaFzQ7/686c-Add-or-Remove-Dependents?node-id=1291-80167&t=ALQ9JbSieaec8CWW-4), we should remove the header.

- page `/current-spouse-marriage-history/0/reason-former-marriage-ended?edit=true`
- [design](https://www.figma.com/design/7W55oNwdVXvXOTI9SaFzQ7/686c-Add-or-Remove-Dependents?node-id=1291-80167&t=ALQ9JbSieaec8CWW-4)

## Tasks
- [ ] Remove radio header level
- [ ] Add unit tests if needed

## Acceptance Criteria
- [ ] All tasks complete
- [ ] All tests passing

**Comment by davidakennedy** [comment]
@fiorella-io When you get a chance, can you validate this ticket for Robin.
